__tests__/endpoint/seed.test.js

```diff
@@ -20,14 +20,33 @@ describe("/api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then((response) => {
-        expect(response.body.topics).toHaveLength(3)
+        expect(response.body.topics).toHaveLength(3);
         response.body.topics.forEach((topic) => {
           expect(typeof topic.slug).toBe("string");
           expect(typeof topic.description).toBe("string");
         });
       });
   });
+});
+
+describe("/api", () => {
+  test("should GET: 200 responds with an object which describes all the endpoints on this API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const parsedResponse = JSON.parse(response.text);
+        const vals = Object.values(parsedResponse);
+        vals.forEach((obj) => {
```

Comment:
One thing to keep in mind when making assertions within a forEach, is that you should also be checking the length before hand. This is because if the array was empty, the code in the forEach wouldn't run. This means that your expect lines wouldn't have a chance to run and cause the test to fail - so keep this in mind :)